load_fixtures.py

```diff
@@ -0,0 +1,18 @@
+from os import chdir, getcwd, listdir, path, system
+
+original_directory = getcwd()
+apps = ['authentication', 'property', 'pages']
+files = []
+
+
+def loaddata(file):
+    if path.splitext(file)[1] == '.json' and file != 'initial_data.json':
+        print(file)
```

Comment:
Remove print statements

Reply:
Thank you @bl4ck4ndbr0wn, I am going to act on this.

Reply:
@bl4ck4ndbr0wn I am done removing the print statements.